patches/server/1027-Fix-MC-255756-shield-break-block-sounds-not-playing.patch

```diff
@@ -0,0 +1,57 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: roro1506HD <roro1506mcpro@gmail.com>
+Date: Fri, 25 Aug 2023 14:42:54 +0200
+Subject: [PATCH] Fix MC-255756 shield break/block sounds not playing
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/LivingEntity.java b/src/main/java/net/minecraft/world/entity/LivingEntity.java
+index e11d7283662834047b2ff81a2fd25a4263792deb..7e02de5ae22abeaad6607c5856e8326538716acb 100644
+--- a/src/main/java/net/minecraft/world/entity/LivingEntity.java
++++ b/src/main/java/net/minecraft/world/entity/LivingEntity.java
+@@ -1482,7 +1482,7 @@ public abstract class LivingEntity extends Entity implements Attackable {
+             boolean knockbackCancelled = this.level().paperConfig().environment.disableExplosionKnockback && source.is(DamageTypeTags.IS_EXPLOSION) && this instanceof net.minecraft.world.entity.player.Player; // Paper - Disable explosion knockback
+             if (flag1) {
+                 if (flag) {
+-                    this.level().broadcastEntityEvent(this, (byte) 29);
++                    this.level().playSound(null, this, SoundEvents.SHIELD_BLOCK, SoundSource.PLAYERS, 0.8F, 0.8F + this.level().random.nextFloat() * 0.4F); // Paper - Fixes MC-255756: Play the sound instead of the entity event
```

Comment:
Part of the comment should be where exactly you got this line (which I assume is from the handleEntityEvent in LivingEntity). Also, I think that first float param should be `1.0F` for event 29.

Reply:
Nice catch, sorry for not paying too much attention on this. And got you for the comment, going to add that